frontend/src/components/editor-page/document-bar/revisions/revision-list-entry.tsx

```diff
@@ -74,6 +75,10 @@ export const RevisionListEntry: React.FC<RevisionListEntryProps> = ({ active, on
         </ShowIf>
         <ShowIf condition={!revisionAuthors.error && !revisionAuthors.loading}>{revisionAuthors.value}</ShowIf>
       </span>
+      <span>
+        <UiIcon icon={IconPersonPlus} className='mx-2' />
+        <Trans fqdn={'Anonymous authors/guests'} />: {revision.anonymousAuthorCount}
```

Comment:
fqdn is no valid attribute for the Trans component.
Use i18nKey but write the value of the property like in line 69.
The actual translation should be put into `frontend/locales/en.json`.

Reply:
Ok, is that alright now?